paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -54,13 +54,13 @@ We provide the results of accuracy and performance measured on Intel(R) Xeon(R)
 
```

Comment:
Please use `INT8 MKL-DNN  post-training quantization` instead of `INT8 MKL-DNN  quantization` as the title.

About `Accuracy Diff`: If the int8 accuracy is dropped, please use the negative value. Otherwise, use positive value.

Reply:
Done.